### PR TITLE
SSL error solved

### PR DESCRIPTION
**Pull request by Jaunter**
I solved an SSL error that threw my script while running.

### CODE REVIEW

File: google_images_download/google_images_download.py [hunk 1]
```diff
@@ -99,7 +99,7 @@ def run(self, item, filename, filename_format='basename', no_clobber=True):
         """
         try:
             req = Request(item, headers={"User-Agent": self.ua.firefox})
-            with urlopen(req) as response, \
+            with urlopen(req, context=ssl._create_unverified_context()) as response, \
```
Comment (rachmadaniHaryono) [inline google_images_download/google_images_download.py:102]: Use try/except for the default method and catch ssl.CertificateError? That way the default is still used until error happened. 

Also add debug logging when catching ssl.CertificateError

**Review by rachmadaniHaryono — CHANGES_REQUESTED** [review]

File: google_images_download/google_images_download.py (hunk 1 above)
Comment (rytoj) [reply google_images_download/google_images_download.py:102]: Do we merge this?

Comment (rachmadaniHaryono) [reply google_images_download/google_images_download.py:102]: yes, just wait for @rytoj

**Comment by rachmadaniHaryono** [comment]
@rytoj, this pr can be closed as #6 already implement this feature

**Comment by rytoj** [comment]
Implemented  [ #6 ](https://github.com/rytoj/google-images-download/pull/6)

**Pull request closed without merging**